docs/content/platform/new_blocks.md

```diff
@@ -42,10 +42,10 @@ Follow these steps to create and test a new block:
    Example:
 
    ```python
-   class Input(BlockSchema):
+   class Input(BlockSchemaInput):
        topic: str  # The topic to get the Wikipedia summary for
 
-   class Output(BlockSchema):
+   class Output(BlockSchemaOutput):
        summary: str  # The summary of the topic from Wikipedia
        error: str  # Any error message if the request fails, error field needs to be named `error`.
```

Comment:
this part of docs also needs updated:
```suggestion
```

Reply:
I feel like our docs are replicated heavily. updated